src/sentry/static/sentry/app/components/events/interfaces/frame.jsx

```diff
@@ -175,11 +175,21 @@ export const Frame = createReactClass({
         ? data.module || data.filename
         : data.filename || data.module;
 
-      title.push(
-        <code key="filename" className="filename">
-          <Truncate value={pathName} maxLength={100} leftTrim={true} />
-        </code>
-      );
+      if (defined(data.absPath)) {
+        title.push(
+          <Tooltip title={data.absPath || ''}>
```

Comment:
Is it going to render an empty tooltip if it's blank?

Reply:
Asl you can just use the `disabled` prop instead of having this if / else condition.

Reply:
:+1: to what @EvanPurkhiser suggested. Using the `disabled` prop on the Tooltip will hide it.